lib/pure/httpclient.nim

```diff
@@ -245,7 +245,8 @@ proc contentLength*(response: Response | AsyncResponse): int =
   ##
   ## A ``ValueError`` exception will be raised if the value is not an integer.
   var contentLengthHeader = response.headers.getOrDefault("Content-Length")
-  return contentLengthHeader.parseInt()
+  result = contentLengthHeader.parseInt()
```

Comment:
Use ``parseSaturatedNatural`` because that cannot raise an overflow exception.

Reply:
Yes, but that changes the documented behaviour:

> A ``ValueError`` exception will be raised if the value is not an integer.

```nim
import parseutils, strutils

var a = "123.45"

var b: int
discard parseSaturatedNatural(a, b)
echo b # ==> 123

var c = parseInt(a)
echo c # ==> ValueError
```

Reply:
Ok...